app/main.py

```diff
@@ -2,0 +1,33 @@
+def copy_file(command: str) -> None:
+    if not isinstance(command, str) or len(command) <= 1:
+        print("Command is incorrect")
+        return
+
+    if command[:2] != "cp":
+        print("Command should start with 'cp'")
+        return
+
+    cmd_list = command.strip().split(" ")
+    if len(cmd_list) <= 2:
+        print("Command 'cp' should contains at least two arguments")
+        return
+
+    if cmd_list[-2] == cmd_list[-1]:
+        print("Warning: arguments are the same")
+        return
+
+    try:
+        open(cmd_list[-1], "r")
+    except FileNotFoundError:
+        pass
+    else:
+        confirm = input("Destination file exists, confirm override (y/n):")
+        while confirm not in ["y", "Y", "n", "N"]:
+            confirm = input("Destination file exists, confirm override (y/n):")
+        if confirm in ["n", "N"]:
+            return
```

Comment:
This part of the code checks if the destination file exists, and if it does, it asks the user for confirmation before overwriting it. This is a good approach, but you should also check if the source file exists before trying to open it. If the source file does not exist, the code should print an error message and return without doing anything else.